components/annonces-section/bodacc.tsx

```diff
@@ -45,6 +45,7 @@ const AnnoncesBodaccSection: React.FC<{
     return (
       <LoadingSection
         title="Annonces BODACC"
+        description="Nous récupérons les annonces de cette structure"
         sources={[EAdministration.DILA]}
       />
     );
```

Comment:
To add an animation, it would need to merge `<LoadingSection />` into the default `<Section />` component